setup.py

```diff
@@ -39,7 +39,7 @@ def read_file(filename):
 
 setup(
     name='ccdb5_ui',
-    version=get_git_version(),
+    version='2.3.3-dev5',
```

Comment:
Not related to your PR, but could we revert? I made this change while struggling to get ccdb deployed to DEV5, and we can switch back now.

```suggestion
    version=get_git_version(),
```